packages/bpk-component-barchart/package.json

```diff
@@ -18,7 +18,7 @@
     "bpk-react-utils": "^2.6.1",
     "bpk-tokens": "^27.0.4",
     "d3-path": "^1.0.5",
-    "d3-scale": "^1.0.5",
+    "d3-scale": "^2.1.0",
```

Comment:
Breaking changes here are related to color schemes (see https://github.com/d3/d3-scale/releases). We should be unaffected.